src/lib/mdns/minimal/tests/CheckOnlyServer.h

```diff
@@ -36,85 +38,188 @@ namespace mdns {
 namespace Minimal {
 namespace test {
 
-class CheckOnlyServer : public ServerBase, public ParserDelegate
+constexpr QNamePart kIgnoreQNameParts[] = { "IGNORE", "THIS" };
+namespace {
+bool StringMatches(const BytesRange & br, const char * str)
+{
+    return br.Size() == strlen(str) && strncmp(str, reinterpret_cast<const char *>(br.Start()), br.Size()) == 0;
```

Comment:
nit: consider using memcmp, since then you do not need  the cast.

Reply:
done in next commit.